src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -621,6 +632,51 @@ public void testLaunchHICase() throws Exception {
     }
   }
 
+  @SneakyThrows
+  private void checkRejectInvalidLaunchCombo(String expectedMsg) {
```

Comment:
Any chance these methods can be re-named assertX? I had to check what the methods did. I always feel if a method like this runs an assert, it should be abundantly obvious even to idiots like me.

Reply:
sure sounds like a good name change